src/screens/WebView/PrivacyPolicies.js

```diff
@@ -1,36 +1,37 @@
 import * as React from 'react';
-import { ActivityIndicator, SafeAreaView, StyleSheet, View } from 'react-native';
-import { WebView } from 'react-native-webview';
+import {ActivityIndicator, SafeAreaView, StyleSheet, View} from 'react-native';
+import {WebView} from 'react-native-webview';
 import Header from '../../components/Header';
-import { withInteractionsManaged } from '../../components/WithInteractionManaged';
+import {withInteractionsManaged} from '../../components/WithInteractionManaged';
+import {COLORS} from '../../utils/theme';
 
 const PrivacyPolicies = ({navigation}) => (
-    <View style={styles.container}>
-      <SafeAreaView>
-      <Header title="Privacy Policies" onPress={() => navigation.goBack()}  />
-      </SafeAreaView>
-      <WebView
-        androidHardwareAccelerationDisabled={false}
-        source={{ uri: 'https://bettersocial.org/privacy' }}
-        showsVerticalScrollIndicator={false}
-        startInLoadingState={true}
-        style={styles.webview}
-        renderLoading={() => (
-          <View style={styles.loading}>
-            <ActivityIndicator size="large" color="#00ADB5" />
-          </View>
-        )}
-      />
-    </View>
-  );
+  <View style={styles.container}>
+    <SafeAreaView>
+      <Header title="Privacy Policies" onPress={() => navigation.goBack()} />
+    </SafeAreaView>
+    <WebView
+      androidHardwareAccelerationDisabled={false}
+      source={{uri: 'https://bettersocial.org/privacy'}}
+      showsVerticalScrollIndicator={false}
+      startInLoadingState={true}
+      style={styles.webview}
+      renderLoading={() => (
+        <View style={styles.loading}>
+          <ActivityIndicator size="large" color={COLORS.holyTosca} />
+        </View>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change of hardcoded color value to a reference from the theme file is a good practice for maintainability and consistency. 



<!-- This is an auto-generated comment by OSS CodeRabbit -->